mvn_setup/tasks-parent/tasks-ui/src/core/MyWork/MyWorkGroup.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from 'react';
+import { Box, useTheme, Typography, IconButton } from '@mui/material';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+
+import Client from '@taskclient';
+import { FormattedMessage } from 'react-intl';
+
+const groupTitles: Record<string, string> = {
+  assigneeOverdue: "Overdue work",
+  assigneeOther: "Assigned to me",
+  assigneeStartsToday: "Starts today"
+}
+
+const StyledTaskItem: React.FC<{ task: Client.TaskDescriptor }> = ({ task }) => {
+  const theme = useTheme();
+  const status = task.status;
+  const groupId = task.assigneeGroupType;
+  const [active, setActive] = React.useState(false);
+
+  function handleMouseOver(){
+    setActive(true)
+  }
+
+  function handleMouseLeave(){
+    setActive(false)
+  }
+
+  function formatTaskDate(date: Date | undefined): string | undefined {
+    return date?.toLocaleDateString();
+  }
+
```

Comment:
The whole component should be re-rendered when a new instance of TaskDescriptor is supplied. So, the useMemo in here doesn't have value.